zScanner/SupportFiles/Extensions/UITextField.swift

```diff
@@ -9,47 +9,17 @@
 import UIKit
 
 extension UITextField {
-    func setBottomBorder(show: Bool = true, color: CGColor = UIColor.white.cgColor, animated: Bool = false, duration: CFTimeInterval = 1) {
+    func setBottomBorder(show: Bool = true, animated: Bool = true, color: UIColor = .primary, duration: CFTimeInterval = 0.333) {
```

Comment:
Most of the system animations are around 0.3 - 0.35s. 
I tend to set default animation time 0.333 just because the number looks good. 
1s is too long for default animation. Animations should be non-disruptive.